site/_less/layouts/action-network-petitions/white-two-column.less

```diff
@@ -139,11 +145,105 @@
 
   .petition-info {
     @media screen and (min-width: @large-phone) {
-      padding: 0 1rem 0 0;
+      padding: 0 2rem 0 0;
     }
   }
 
+  [for="opt-in"] {
+    background-color: white;
+    color: @blue-800;
+    padding: 2rem 5px;
+  }
   .an-footer-cta {
     display: none;
   }
 }
+
+
+ .petition blockquote p{
+    font-size: 0.8em;
+    color: #ecedfb;
+    line-height: 2rem;
+  }
+
+  main.white-two-column h2{
+    margin-bottom: 0.5em;
+  }
+
+  .white-two-column [for="opt-in"]{
+    color: #8889a0;
+  }
+
+  .petition-info h2{
+    font-weight: 400;
+  }
+
+  .petition [type=email], .petition [type=text]{
+    padding: 1.5rem;
+    font-size: 18px;
+  }
+
+input[placeholder], [placeholder], *[placeholder] {
+  color: #4f4f69 !important;
```

Comment:
We don't use `!important` in our css. This would probably be best written as

```less
input, textarea {
  &::placeholder {
    color: #4f4f69;
  }
}
```

(also all colors should be variables, _ideally_ reusing colors when available. so instead of `#4f4f69` above, creating a new variable in variables.less and inserting _that_ above.)